web3data/handlers/market.py

```diff
@@ -215,7 +215,7 @@ def order_book(self, pair: str, **kwargs) -> Dict:
             params=kwargs,
         )
 
-    def uniswap_liqudity(self, pair: str, **kwargs) -> Dict:
+    def uniswap_liquidity(self, pair: str, **kwargs) -> Dict:
```

Comment:
Amberdata is going to have a new endpoint coming out soon that will have more info for liquidity and include other exchanges. Consider deprecating this?

Reply:
Will the endpoint be phased out at a certain date? I can log a warning and remove it in a new release after it's removed. :slightly_smiling_face:

Reply:
I think for now we will support it until something comes along to break it. A warning would be nice, but not required.

Sorry for the late reply!

Reply:
I added a warning in version `0.1.7` which just landed. 👍🏼 Good suggestion!